Kinetic_space_partition/examples/Kinetic_space_partition/CMakeLists.txt

```diff
@@ -1,39 +1,24 @@
 # Created by the script cgal_create_CMakeLists.
 # This is the CMake script for compiling a set of CGAL applications.
 
-cmake_minimum_required(VERSION 3.1...3.15)
+cmake_minimum_required(VERSION 3.1)
```

Comment:
It should be

```cmake
cmake_minimum_required(VERSION 3.1...3.23)
```

like in the rest of CGAL (`master`). That tells CMake that CMake policies up to those introduced by CMake 3.23 were tested and should be set to `NEW`.

Reply:
As a side note:
```
Alpha_wrap_3/benchmark/Alpha_wrap_3/CMakeLists.txt:cmake_minimum_required(VERSION 3.1...3.20)
SMDS_3/examples/SMDS_3/CMakeLists.txt:cmake_minimum_required(VERSION 3.1...3.20)
SMDS_3/test/SMDS_3/CMakeLists.txt:cmake_minimum_required(VERSION 3.1...3.20)
```

Reply:
> As a side note:
> 
> ```
> Alpha_wrap_3/benchmark/Alpha_wrap_3/CMakeLists.txt:cmake_minimum_required(VERSION 3.1...3.20)
> SMDS_3/examples/SMDS_3/CMakeLists.txt:cmake_minimum_required(VERSION 3.1...3.20)
> SMDS_3/test/SMDS_3/CMakeLists.txt:cmake_minimum_required(VERSION 3.1...3.20)
> ```

Indeed, from time to time I try to bump the "max" version of CMake. When package are merged into CGAL, they tend to use the version numbers at the time their branch was forked, and not the one at the time they were merged. The package `Alpha_wrap_3` was introduced in CGAL-5.5 and `SMDS_3` in CGAL-5.6. The bump to `3.1...3.23` was during the release cycle of 5.5 (PR https://github.com/CGAL/cgal/pull/6558).

As far as I know, the only policy introduced since CMake 3.20 that is relevant for CGAL is [CMP0153](https://cmake.org/cmake/help/latest/policy/CMP0153.html), that was dealt with by PR https://github.com/CGAL/cgal/pull/7963 (merged in all branches of CGAL).

Reply:
@lrineau  I assume you will correct this and bring these 3 cases up to date as well?

Reply:
If the `CMakeLists.txt` keeps using `cmake_minimum_required(VERSION 3.1)`, then that will trigger warning in the testsuite, and maybe errors like what appeared in last night testsuite:
```
Unable to find executable: of
Unable to find executable: source_dir
Failed to change working directory to /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Kinetic_surface_reconstruction_Examples/__exec_test_dir : No such file or directory
Unable to find executable: of
Unable to find executable: of
Failed to change working directory to /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Kinetic_surface_reconstruction_Examples/__exec_test_dir : No such file or directory
Unable to find executable: of
Failed to change working directory to /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Kinetic_surface_reconstruction_Examples/__exec_test_dir : No such file or directory
Unable to find executable: source_dir
Unable to find executable: of
Unable to find executable: of
Failed to change working directory to /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Kinetic_surface_reconstruction_Examples/__exec_test_dir : No such file or directory
Unable to find executable: source_dir
Unable to find executable: of
Unable to find executable: of
Failed to change working directory to /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Kinetic_surface_reconstruction_Examples/__exec_test_dir : No such file or directory
Unable to find executable: of
Failed to change working directory to /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Kinetic_surface_reconstruction_Examples/__exec_test_dir : No such file or directory
Unable to find executable: of
Failed to change working directory to /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Kinetic_surface_reconstruction_Examples/__exec_test_dir : No such file or directory
Unable to find executable: source_dir
Unable to find executable: of
Unable to find executable: of
Failed to change working directory to /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Kinetic_surface_reconstruction_Examples/__exec_test_dir : No such file or directory
Unable to find executable: of
Failed to change working directory to /home/cgal_tester/build/src/cmake/platforms/ArchLinux-CXX14/test/Kinetic_surface_reconstruction_Examples/__exec_test_dir : No such file or directory
```

I am not sure why there was errors, actually.